packages/testsuite/cypress/e2e/mail/test-configuration-subsystem-mail-credential-reference.cy.ts

```diff
@@ -0,0 +1,142 @@
+describe("TESTS: Configuration => Subsystems => Mail", () => {
+  const address = [
+    "subsystem",
+    "mail",
+    "mail-session",
+    "my-mail-session",
+    "server",
+    "smtp",
+  ];
+
+  const outBoundSocketBinding = {
+    name: "custom-outbound-socket-binding",
+    host: "localhost",
+    port: "15099",
+  };
+
+  const configurationFormId = "mail-server-form";
+  const mailServerAddFormId = "mail-server-add-form";
+  const credentialsReferenceFormId = "mail-server-credential-reference-add";
+
+  const mailSessionTableId = "mail-server-table";
+  const mailSessionOutboundSocketType = "SMTP";
+
+  const formColumNameClearText = "clear-text";
+  const formColumValueClearText = "ClearTextValue";
+  const credentialsReference = "credential-reference";
+  const formColumNameName = "name";
+  const formColumValueName = "my-mail-session";
+  const formColumNameFrom = "from";
+  const formColumValueDefaultFrom = "DefaultFromValue";
+  const formColumNameJNDI = "jndi-name";
+  const formColumValueDefaultJNDI = "java:jboss/mail/MyJNDI";
+
+  const formColumNameSSL = "ssl";
+  const formColumNameTLS = "tls";
+
+  const formColumNameOutbound = "outbound-socket-binding-ref";
+  const formColumValueOutbound = "custom-outbound-socket-binding";
+
+  const formColumNameUserName = "username";
+  const formColumValueUserName = "Custom username";
+
+  const formColumNamePassword = "password";
+  const formColumValuePassword = "superSecretPassword";
+
+  let managementEndpoint: string;
+
+  before(() => {
+    cy.startWildflyContainer()
+      .wait(2000)
```

Comment:
Using a fix value for `wait` seems a bit dangerous as test execution environments differs. I suggest to have a constant that can be adjusted by a system property, like in WildFly TS. @spriadka WDYT please?

There are 3 `wait` calls used in this PR. As the values differs we can have multiple constants/functions, e.g. `waitForServerStart` and `waitForPageRendered`.

Reply:
I review all 3 waits in this PR and it seems they are not required now.